_targets.R

```diff
@@ -24,29 +25,24 @@ targets_list <- list(
     split_lake_centroids(centroids_sf_rds)
   ),
 
-  # Convert the lake centroids to some kind of polygon for querying GDP
-  tar_target(
-    query_polys_sf,
-    centroids_to_poly(query_centroids_sf)
-  ),
-
   # Convert the sf polygons into a geoknife-ready format
```

Comment:
Just looks like you missed updating this comment here to refer to the centroids, rather than the lake polygons

Reply:
But then again we'll be tweaking this to instead use the grid cell centroids anyhow, so will change shortly